doc/1-intro/faq.md

```diff
@@ -0,0 +1,138 @@
+
+### is there an iOS build?
+
+nope.
+
+### will Furnace ever have a piano roll or DAW interface?
+
+there are no plans for this.
+
+### I've lost my song – what do I do?
+
+Furnace keeps backups of the songs you've worked on before. go to **file > restore backup**.
+
+## tracking
+
+### how do I compose in 6/8, 5/4, or other time signatures?
+
+set the pattern length to a multiple of the number of beats (6 or 5 as mentioned above). don't forget to change the row highlight values to match!
+
+### how do I track triplets, quintuplets, or other tuplets?
+
+there are two common methods:
+- use delay commands (`EDxx`) to offset notes into their correct places. this is good for the occasional set of tuplets, but if you expect to use a lot of them...
+- plan ahead for the song to have them by making your pattern length a multiple of that number. remember to adjust row highlight values to match.
+
+depending on the tempo of the song, it may only be possible to get perfectly even tuplets by changing the tick rate. mind that the result may not be authentic to hardware.
+
+### why do certain notes not play low enough, high enough, or in tune?
+
+each chip has its own set of limitations regarding what frequencies it can play. if these limits are likely to be found in normal tracking, they'll be mentioned in [that chip's documentation](../7-systems/).
+
+### does Furnace support tunings other than twelve tone equal temperament?
+
+no. both tracker and format are built on that scale and tuning.
+
+### can I add effects to the output like EQ or reverb?
+
+not yet, but it's in the early planning stages.
+
+## chips
+
+### will Furnace support the Sony PlayStation?
+
+it's in the plans, with no target date.
+
+### will Furnace support the Sega Saturn?
+
+it also is in the plans, with no target date.
+
+### will Furnace support more Yamaha FM chips?
+
+once a chip has an available emulation cores written in C++ and licensed under GPL2, it'll be considered for addition.
+
+### will Furnace support the Nintendo 64?
+
+the N64 lacks any form of audio synthesizer chip. many games use MIDI or XM or other such formats internally, but everything is mixed in software and sent to a simple stereo DAC.
+
+### will Furnace support this obscure PCM-only chip?
+
+probably not, as with very few exceptions these are effectively all the same.
+
+### will Furnace support the Roland MT-32?
+
+no. MT-32 is used with MIDI in 99.999% of situations. it lacks a direct register interface.
+
+also, Furnace is not a MIDI tracker....
+
+## importing
+
+### will Furnace import MIDI files?
+
+nope. Furnace is not a MIDI tracker.
+
+### why does this imported file sound wrong?
+
+Furnace makes every attempt to import non-Furnace files as accurately and completely as possible, but there are fundamental differences between formats that cannot be directly translated. an import from a non-Furnace format should always be considered the starting point of a conversion, not a final product.
```

Comment:
Remove "every attempt". Remove "non-Furnace". Remove "as accurately and completely as possible".

The truth is that import could be better...